.eslintignore

```diff
@@ -0,0 +1 @@
+test/async-function-tests.js
```

Comment:
Thank you for bringing this up! I was having a mental block on this. Turns out that I just forgot that I have to setup `ecmaVersion`.  In order to ensure that the rest of the files do not allow all of ES2017, I created a subdir (`test/es2017`) for this one file.